addons/stock/report/report_stock_forecasted.py

```diff
@@ -107,13 +109,18 @@ def _get_report_data(self, product_template_ids=False, product_variant_ids=False
             res['virtual_available'] = sum(product_variants.mapped('virtual_available'))
         res.update(self._compute_draft_quantity_count(product_template_ids, product_variant_ids, wh_location_ids))
 
-        res['lines'] = self._get_report_lines(product_template_ids, product_variant_ids, wh_location_ids)
+        res['lines'] = self._get_report_lines(product_template_ids, product_variant_ids, wh_location_ids, move_to_match_id)
         return res
 
-    def _prepare_report_line(self, quantity, move_out=None, move_in=None, replenishment_filled=True, product=False, reservation=False):
+    def _prepare_report_line(self, quantity, move_out=None, move_in=None, replenishment_filled=True, product=False, reservation=False, move_to_match_id=False):
         timezone = self._context.get('tz')
         product = product or (move_out.product_id if move_out else move_in.product_id)
         is_late = move_out.date < move_in.date if (move_out and move_in) else False
+
+        move_to_match_id = self.env.context.get('move_to_match_id')
```

Comment:
You pass it in the args of the method, but you still take in it from the context. 
IMO, you should remove it from args of methods `_prepare_report_line` and `_get_report_lines`?